pkgs/servers/rmfakecloud/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
```

Comment:
two notes:
 - there's been a arms-long list of discussions about buildGoModules vs. buildGoPackage because buildGoModules duplicates all go dependencies/fixed output derivations are frowned upon, see https://github.com/NixOS/nixpkgs/issues/84826 
That being said, I don't think we should block on this as the new shiny solution (gomod2nix) isn't merged in nixpkgs yet
 - if you update to v0.0.6 (see next comment) you might need to switch to buildGo117Modules, I wasn't able to build  the latest release with go 1.16, but that might be related to the UI that I include in my local package

Reply:
I'm not sure of how to migrate to `buildGoPackage`. If it's still acceptable until all the tools are merged, let's keep it that way for now.

The package seems to build just fine with the default version of the channel (1.16) for me (without the web UI for now).